src/history/history-entry.enity.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '../users/user.entity';
+import { Note } from '../notes/note.entity';
+
+@Entity()
+export class HistoryEntry {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @ManyToOne((_) => User, (user) => user.historyEntries, {
+    onDelete: 'CASCADE',
+  })
+  user: User;
+
+  @ManyToOne((_) => Note, (note) => note.historyEntries, {
+    onDelete: 'CASCADE',
+  })
+  note: Note;
+
+  @Column()
+  pinStatus: boolean;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
```

Comment:
This seems to be missing from the PlantUML schema

Reply:
I'll add this.

Reply:
done